### DISCUSSION TITLE
How much time does it take to transfer a Github repo ?

### DISCUSSION THREAD

**Discussion by MaximeLepageUQAC** [community/community#44732] [Other Feature Feedback, Questions, & Ideas] [answered]
So I made a new account and I want to transfer one of my repository from my old account, but when I got in 
settings->Danger Zone->Transfer repository
Everything seems to be working, now I have a button "Abort Transfer" that shows up, but it's been hours now. Is there something else I am supposed to do ? How much more time should I be waiting ?

**Comment by radar-blip** [comment, marked as answer]
Have you checked the new owners (yours) email for a confirmation email.

Check article here: https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#about-repository-transfers

**Reply by MaximeLepageUQAC** [reply]
Yes I've looked at my emails in either accounts and, there is none concerning this....

**Comment by mdcodelab** [comment]
When you initiate a repository transfer on GitHub, it typically takes a few minutes for the transfer to complete. If it's been several hours and the transfer still hasn't completed, there may be an issue with the transfer. You can try clicking the "Abort Transfer" button and then re-initiating the transfer.

If the problem persists, you can contact GitHub Support for further assistance. They will be able to check the status of the transfer and help you troubleshoot any issues.

**Comment by Shalakahk** [comment]
Try clicking somewhere else. the page might just need refreshing to update the info. When I initiated transfer it showed same thing. I waited for about 30 minutes and then clicked on my Dashboard (to see if I see anything different) It refreshed and showed the repository gone from the account and the transferred repository on the destination account. Transfer worked!

**Comment by b92273667-stack** [comment]
I don't understand what a repository account is
